### PR TITLE
 update cn basics translation part2

### PR DESCRIPTION
**Pull request by redsun**
@Jinxiansen check~

### CODE REVIEW

**Comment by Jinxiansen** [comment]
@redsun 
Thanks for your update, I'm in vacation this week. 
I will review this PR next week.

File: docs/basics/environment.zh.md [hunk 1]
```diff
@@ -87,25 +85,25 @@ print(foo) // String?
...
+当项目被 cloned 到新计算机时，已经带有正确的值的 `.env` 模板可以被复制。
 
 ```sh
 cp .env .env.development
 vim .env.development
 ```
 
 !!! warning "警告"
-    带有敏感信息(如密码)的Dotenv文件不应提交给版本控制。
+    带有敏感信息(如密码)的 Dotenv 文件不应提交给版本控制。
```
Comment (Jinxiansen) [inline docs/basics/environment.zh.md:104]: **Dotenv** 建议为 **.env**

File: docs/basics/environment.zh.md [hunk 2]
```diff
@@ -62,22 +61,21 @@ export FOO=BAR
...
-当在Xcode中运行应用程序时，你可以通过编辑 `Run` scheme来设置环境变量。
+当在 Xcode 中运行应用程序时，你可以通过编辑 `Run` scheme 来设置环境变量。
 
 ## .env (dotenv)
 
-Dotenv文件包含一个键值对列表，这些键值对将自动加载到环境中。这些文件使配置环境变量变得很容易，而不需要手动设置它们。
+Dotenv 文件包含一个键值对列表，这些键值对将自动加载到环境中。这些文件使配置环境变量变得很容易，而不需要手动设置它们。
 
-Vapor 将在当前工作目录中查找dotenv文件。如果你使用Xcode，确保通过编辑 `Run` scheme 设置工作目录。
+Vapor 将在当前工作目录中查找 dotenv 文件。如果你使用 Xcode，确保通过编辑 `Run` scheme 设置工作目录。
```
Comment (Jinxiansen) [inline docs/basics/environment.zh.md:70]: dotenv → .env

File: docs/basics/environment.zh.md [hunk 3]
```diff
@@ -87,25 +85,25 @@ print(foo) // String?
 !!! info "信息"
     在 `.env` 文件中指定的变量不会覆盖进程环境中已经存在的变量。
 
-在`.env`旁边，Vapor 还将尝试为当前环境加载一个dotenv文件。例如，在 `development` 环境中，蒸汽将加载 `.env.development`。特定环境文件中的任何值都将优先于 `.env` 文件内的值。
+除了 `.env`，Vapor 还将尝试为当前环境加载一个 dotenv 文件。例如，在 `development` 环境中，Vapor 将加载 `.env.development`。特定环境文件中的任何值都将优先于 `.env` 文件内的值。
```
Comment (Jinxiansen) [inline docs/basics/environment.zh.md:88]: dotenv → .env.environment

Comment (redsun) [reply docs/basics/environment.zh.md]: ok 了

**Comment by redsun** [comment]
这个是没有合并么？

**Review by Jinxiansen — APPROVED** [review]

**Pull request merged**